tests/qa_automation/xfstests_device.pm

```diff
@@ -55,6 +55,8 @@ sub dev_create_partition {
     $test_partition = "/dev/vda" . $test_partition_id;
 
     type_string "parted --script --machine -l\n", 5;
+    #workaround bsc#1072549
+    type_string "umount /home\n", 5;
```

Comment:
better to use `record_soft_failure` with the bug as reference to present a false on test

Reply:
Refer to the bug will not fix, and this change will stay, so don't need to change to soft failure then...

Reply:
ah, just seen that is 'invalid' and gave the workaround :)